tests/assets/disable_polling_reset_2_replicas.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+echo -e "Cloning repo"
```

Comment:
Suggestion: Add `\n` in every `echo`. The output in the `cy.log` is not formatted. <-- May be you can implement this suggestion in next PR with similar test.

Reply:
Thanks, but that was the intention with `echo -e`. It does not format it anyway there.